src/Backend/Modules/MediaLibrary/Js/MediaLibraryHelper.js

```diff
@@ -1478,5 +1479,21 @@ jsBackend.mediaLibraryHelper.templates =
     }
 };
 
+jsBackend.mediaLibraryHelper.modalSelection = {
+    init: function () {
+        $('tr[data-direct-url] a').on('click', this.selectItemAndSendToParent);
+    },
+
+    selectItemAndSendToParent: function () {
+        // @todo: improve this selector
```

Comment:
did you improve it

Reply:
I didn't, but now I did.

Fixed, see 0ee8d807.

By changing the way the datagrid is being generated, I can avoid that nasty function call `.parent().parent()` 🙃